### DISCUSSION TITLE
Di quale switches ho bisogno per le mie luci?

### DISCUSSION THREAD

**Discussion by H4Z4RD-H42** [indomus/forum#2306] [Hardware & Firmware]
Premetto che ho letto sia le vostre che altre guide ed ho visto vari video, ma purtroppo quando si parla di impianti elettrici ho sempre grosse difficoltà a capire tutti i collegamenti e come gira la tensione... Un'idea me la sono fatta, ma ho troppe domande senza risposta e prima di comprare 5-6 switches vorrei chiedere a voi che sicuramente ne capite di più.

Ho aperto svariate scatole di interruttori in giro per casa e **ho notato che hanno tutte il neutro**. Inoltre alcuni sono deviatori (e io qui già mi perdo).

Quello che vorrei fare è semplice: rendere tutti gli interruttori che accendono le luci principali delle stanze, smart, quindi non parlo di lampade da terra o da comodino, ma del lampadario/plafoniera. Sono tutte led e nessuna dimmerabile (non mi interessa dimmerarle). Nota fondamentale, cerco switches ZigBee perchè ho problemi di copertura in alcune zone che vorrei colmare, quindi devono fare anche da router (specifico questa ultima frase perchè ho letto la recensione degli ultimi Sonoff ZBMINI che stanno per uscire, ma che non fanno da repeater).

Ho fatto delle foto a 3 scatole aperte, questa per esempio è della camera da letto:
![image](https://user-images.githubusercontent.com/6650832/215284006-fca77cdf-a366-4ead-97df-b1334b11f391.png)

Questa invece della cameretta (ma è uguale all'altra, solo che la stessa luce la posso accendere anche da un altro punto):
![image](https://user-images.githubusercontent.com/6650832/215284044-cccb2b1a-f0c8-463a-8e7b-7547e614b5dc.png)

Questa invece è la scatola del soggiorno, dove, da sinistra a destra, ci sono 3 interruttori per accendere le luci principali e che sono quelli che vorrei rendere smart e in modo indipendente l'uno dall'altro, poi c'è uno spazio vuoto, e poi altri interruttori delle luci esterne che per il momento non mi interessano ma in futuro vorrei domotizzare anche quelli.
![image](https://user-images.githubusercontent.com/6650832/215284201-d029b540-5c07-47f5-ad17-833087fca55d.png)

Bene o male anche le altre stanze rimanenti sono fatte alle stessa maniera.. Se gli switches non dovessero entrare in qualche scatola, non ho problemi a inserirli a monte nelle scatole di derivazione.

Quindi... **quali switches fanno al caso mio?** Con neutro suppongo.. O posso comprare anche quelli senza neutro e basta che il filo neutro non lo collego? Poi il carico andrebbe bene lo stesso? E altre domande che non sto nemmeno a fare perchè si, è vero, probabilmente la trovo in rete la risposta, ma se poi non capisco lo stesso, o meglio, non riesco ad applicare quanto appreso al mio impianto elettrico di casa, mi ritrovo da capo a dodici..

**Comment by diaz66** [comment]
Vanno bene entrambi. Io ne ho uno senza neutro , molto piccolo è sta benissimo dentro la scatola dell' interruttore. Lo utilizzo con Home Assistant e ZigBee2mqtt però , da quello che ho visto, sembra che quelli senza neutro **non funzionano come router.** da verificare comunque prima di acquistare.

Le luci che si accendono da 2 punti hanno per forza 2 deviatori (da tre punti 2 deviatori e un commutatore in mezzo).  Un deviatore ha tre fili, un commutatore 4, un interruttore 2. Ecco un esempio di collegamento con 2 deviatori:
![image](https://user-images.githubusercontent.com/32227980/215290475-c4c15e44-a4c2-47ae-a117-b0ad3b48d489.png)
mentre da un punto solo un interruttore.
![image](https://user-images.githubusercontent.com/32227980/215290535-6b29561c-c83a-447c-aba8-68554fc5f357.png)

Indispensabile un cacciavite cercafase ed attrezzi adatti ad operare in tensione , **occorre sapere bene dove mettere le mani...la sicurezza prima di tutto, si rischia di farsi male sul serio.**

**Reply by Scialla** [reply]
dalle foto, non mi sembra che tu abbia il neutro, poi magari non lo hai messo nelle foto..

**Reply by H4Z4RD-H42** [reply]
Questo chiarisce quanto sono ferrato in materia...
Sono propenso a comprare 2 [ZBMINI](https://sonoff.tech/product/diy-smart-switches/zbmini/) da mettere nelle scatole di derivazione in punti strategici in modo che mi ripetono il segnale proprio dove mi serve e 3 [ZBMINI-L](https://sonoff.tech/product/diy-smart-switches/zbmini-l/) invece direttamente nelle scatole degli interruttori, il neutro ce lo porto io.

Comunque mi sono informato e, come dice @diaz66, della famiglia Sonoff, solo il ZBMINI funge da router, tutti gli altri no.
